replications/service_test.go

```diff
@@ -19,6 +19,10 @@ import (
 	"go.uber.org/zap/zaptest"
 )
 
+//go:generate go run github.com/golang/mock/mockgen -package mock -destination ./mock/validator.go github.com/influxdata/influxdb/v2/replications ReplicationValidator
+//go:generate go run github.com/golang/mock/mockgen -package mock -destination ./mock/bucket_service.go github.com/influxdata/influxdb/v2/replications BucketService
+//go:generate go run github.com/golang/mock/mockgen -package mock -destination ./mock/queue_management.go github.com/influxdata/influxdb/v2/replications DurableQueueManager
```

Comment:
It's helpful to have these here - I'll try to keep this pattern going when I see opportunities for it.